frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -181,7 +183,13 @@ export const CreateStorageSystemFooter: React.FC<CreateStorageSystemFooterProps>
           type: 'wizard/setStepIdReached',
           payload: state.stepIdReached <= stepId ? stepId + 1 : state.stepIdReached,
         });
-        onNext();
+        stepName === StepsName(t)[Steps.ReviewAndCreate]
+          ? history.push(
+              `/k8s/ns/${CEPH_STORAGE_NAMESPACE}/${referenceForModel(
```

Comment:
We want to send there here or in the list page inside ODF?

Reply:
Once they are ready with dynamic plugin integration, we can change that.